python/lsst/ts/externalscripts/auxtel/build_pointing_model.py

```diff
@@ -162,8 +165,16 @@ async def configure(self, config):
 
         self.config = config
 
+        # Instantiate BestEffortIsr
+        self.best_effort_isr = self.get_best_effort_isr()
+
         self.configure_grid()
 
+    def get_best_effort_isr(self):
+        # Isolate the BestEffortIsr class so it can be mocked
+        # in unit tests
+        return BestEffortIsr(self.config.datapath, repodirIsGen3=True)
```

Comment:
That's fine too, but there are other reasons still to create this with a butler, rather than a path. I really think this is worth doing - it's the same pattern as this, and means you have a butler to hand if you ever need one, not to mention it means that that dependency is clear and upfront rather than hidden inside the (somewhat weird) BestEffortIsr object.

Reply:
That does sounds like a reasonable thing to do though, I have no idea what it would look like..

Reply:
It's as easy as making a butler, and then changing that line to `self.best_effort_isr = BestEffortIsr(butler=butler)` 🙂